src/Items/ItemBow.h

```diff
@@ -69,17 +69,12 @@ class cItemBowHandler :
 		}
 
 		// Create the arrow entity:
-		cArrowEntity * Arrow = new cArrowEntity(*a_Player, Force * 2);
-		if (Arrow == nullptr)
+		auto ArrowPtr = cpp14::make_unique<cArrowEntity>(*a_Player, Force * 2);
+		auto Arrow = ArrowPtr.get();
+		if (!Arrow->Initialize(std::move(ArrowPtr), *a_Player->GetWorld()))
```

Comment:
Man, this code has some danger potential. I was just typing "`ArrowPtr->Initialize(...)` should work", but then I realized, what would get called first, the `std::move` or the indirection?

We need to come up with a better interface for initialization (after this PR is merged).